lib/chain/src/ChainTypes.js

```diff
@@ -97,7 +98,10 @@ ChainTypes.operations = {
     bid_collateral: 45,
     execute_bid: 46,
     asset_claim_pool: 47,
-    asset_update_issuer: 48
+    asset_update_issuer: 48,
+    htlc_create:49,
+    htlc_redeem:50,
+    htlc_extend:51
```

Comment:
According to https://github.com/bitshares/bitshares-core/blob/master/libraries/chain/include/graphene/chain/protocol/operations.hpp#L99-L104,

* 49 create
* 50 redeem
* 51 redeemed (virtual)
* 52 extend
* 53 refund (virtual)

Reply:
okay, haven't seen that